api/NuageAPI_test.go

```diff
@@ -91,6 +91,10 @@ func execCMDOnRemoteHost(cmd string, host string) error {
 func cleanup(vrsConnection VRSConnection, vmInfo map[string]string) error {
 
 	var err error
+	err = vrsConnection.DeregisterForPortUpdates(vmInfo["entityport"])
```

Comment:
Can we remove this from cleanup function and instead use a defer statement after the RegisterForPortUpdates ? That will make the API use much clearer.